.travis.yml

```diff
@@ -24,11 +24,19 @@ before_install:
   - sudo apt-get install -y g++-4.9
   - sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-4.9 60 --slave /usr/bin/g++ g++ /usr/bin/g++-4.9
   - sudo apt-get install -y gnupg2
+  - sudo apt-get install -y git
+  - sudo add-apt-repository ppa:longsleep/golang-backports -y
+  - sudo apt-get update
+  - sudo apt-get install -y golang-1.12
+  - export PATH="/usr/lib/go-1.12/bin:$PATH"
+  - export GOROOT=/usr/lib/go-1.12
+  - go version
 install:
   - pip3 install -r requirements.txt
   - pip3 install coverage
 
 script:
+  - make build_helm_binding
```

Comment:
move this after `make test` below.
Helm bindings will change less often that normal changes that "make test" should cover. So it's good to fail first on the make test, then on helm bindings. :)

Reply:
@uberspot 
I understand the rationale. If I want to test helm binding as well, then should I do:
`make test` followed by `make build_helm_binding` and then `make test_helm_binding` where I only test the helm binding? Or should I include the testing of helm_binding in the `make build_helm_binding` step?

Reply:
You can have `build_helm_binding` and `test_helm_binding` as separate build steps, no problems there.
Just put `make test` first in the order so it fails early then follow up with the other two build steps. :)

Reply:
Ah, I just got that you are running this build step before the test because the test also tests the generate library. 
Splitting these two test_helm_binding might be making it unnecessarily complex. Let's skip this for now. :)